components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -1550,20 +1555,32 @@ $candidateTargetPaths = @(
     "$PLAN_CONTEXT\$pkg_target\plan.ps1",
     "$PLAN_CONTEXT\habitat\$pkg_target\plan.ps1"
 )
-foreach($place in $candidateTargetPaths) {
-    if(Test-Path $place) {
-        $paths += $place
-        $script:PLAN_CONTEXT = (Split-Path $place -Parent)
+foreach($path in $candidateTargetPaths) {
```

Comment:
As a point of clarity, if we're checking the `$candidateTargetPaths` first, it would make the code a bit easier to read and understand by declaring that array before the `$candidatePaths` one, just to maintain parallel structure.